public/video-ui/src/components/VideoUsages/VideoUsages.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {fetchUsages} from '../../services/capi';
+
+class VideoUsages extends React.Component {
+
+  state = {
+    usages: undefined
+  }
+
+  updateUsages(videoId) {
+
+    if (!videoId) {
+      return this.setState({
+        usages: undefined
```

Comment:
It depends on the situation, undefined in this case means we've never fetched a state, [] means we've fetched the state and there's no usages.